src/screens/Migration/styles.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import { MediaQueryStyleSheet } from 'react-native-responsive';
+import GlobalStyles from '../../global/Styles';
+
+const styles = MediaQueryStyleSheet.create({
```

Comment:
We have global styles under `src/global/Styles.js` which contains a lot of styles you need (e.g. bodyContainer). Also we have `Title` component for screen title. It's not too important right now, but it's always better to keep everything in order right away.

Reply:
Ok, I changed it @florianlenz